server/app/models/place.js

```diff
@@ -16,7 +16,7 @@ var PlaceSchema = new Schema({
     required: true
   },
   photos: [String],
-  likes: [Schema.Types.ObjectId],
+  likes: [String],
```

Comment:
Because, it's not important. We do not need save here ObjectID, because there are no any populating. I think - string is a good variant